packages/ros-translator/ros_translator/typescript/test/test.sh

```diff
@@ -5,6 +5,6 @@ cd $(dirname $0)
 rm -rf out
 PIPENV_MAX_DEPTH=6 pipenv run python -m ros_translator -t=typescript -o=out ros_translator_test_msgs
 echo 'test build'
-npx tsc --noEmit
```

Comment:
Can you revert this? the `$(pwd)/tsconfig.json` shouldn't be needed since we already `cd` into the correct directory

Reply:
Done, https://github.com/open-rmf/rmf-web/pull/613/commits/02a8fdf600b13c130c56e4d4ead40464a166df12